megamek/src/megamek/server/Server.java

```diff
@@ -33592,12 +32377,11 @@ private boolean checkBuildingCollapseWhileMoving(Building bldg,
         } // Handle the next entity
 
         // If we found any infantry, add a line to the phase report.
-        if (foundInfantry) {
+        if (foundInfantry) { // TODO : Windchild this is always false?
```

Comment:
This is always false according to my IDE, is that meant to have happened or is it a bug?

Reply:
My guess is that it should be set to true just after the Compute#isInBuilding check, which leads to reports generated. On the other hand, there's a new line added after the damage report if the infantry takes damage, and I don't think we need two blank lines. But there's no blank line between units if one of them takes no damage. So it's not too consistent. I would put a blank line in at the end if the block that starts with if (Compute.isInBuilding... and remove foundInfantry.